guides/developer/provider-setups/aws-dev-setup.md

```diff
@@ -0,0 +1,291 @@
+---
+layout: single
+title: "Getting Set Up for Spinnaker Development on AWS"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+This guide describes how a developer can set up a [Local Git deployment](/setup/install/environment/#local-git) of Spinnaker on Amazon EC2.
+
+In this setup, Spinnaker runs on an EC2 instance. Code is edited on the laptop and then synced to the EC2 instance for testing.
+
+
+# Configure laptop
+
+1. Create forks of all the [Spinnaker microservices](/reference/architecture/#spinnaker-microservices) on GitHub.
+2. Clone them onto your laptop in a dedicated `/spinnaker` directory. This will simplify syncing them in a batch during development.
+3. Set up [Intellij](/guides/developer/getting-set-up/#intellij) for Spinnaker as specified.
+
+
+# Configure development region
+
+1. Provision some roles for Spinnaker. The specific permissions necessary for each role may vary depending on what AWS providers you wish to use, but in general you will need:
+
+    * `SpinnakerDevInstanceRole`: The instance role to be used by your Spinnaker developer instance.
+        * Trust policy should include `ec2.amazonaws.com`
+        * Attached policy permissions should include:
+            * Amazon S3 full access (for persistent storage)
+            * `ec2:DescribeRegions` + `ec2:DescribeAvailabilityZones`
+            * The ability to assume the `SpinnakerManagedRole` (described below)
+            * Read-only access to Amazon Elastic Container Registry, if using private docker images.
+
+    * `SpinnakerManagedRole` : The role Spinnaker will assume to interact with your AWS resources. See the [AWS Setup Guide](/setup/install/providers/aws/) for more details.
+
+2. In the AWS console, in the region where you want your development instance to reside, [create a key pair](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html#having-ec2-create-your-key-pair). 
+
+3. In the default VPC of the same region, [create a security group](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_SecurityGroups.html#CreatingSecurityGroups) named "SpinnakerDev" and [add a rule](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_SecurityGroups.html#AddRemoveRules) allowing port 22 inbound for a restricted set of IPs (for example, corporate firewall ranges). 
+
+4. Then, provision a development instance that uses these resources:
```

Comment:
This sentence promises to introduce a list (unordered) of resources, and the list is indeed unordered, but it looks to me more like a procedure. 
"_Navigate_ to the EC2 launch instance wizard."
"_Select_ an Ubuntu Server...machine image."

I think it does need to be a procedure, so maybe rewrite this sentence introducing it?